slimmage.js

```diff
@@ -138,11 +138,14 @@
             }
         }
 
+  
+
         //3. Find images with data-slimmage and run adjustImageSrc.
         var totalImages = 0;
         var images = s.nodesToArray(w.document.getElementsByTagName("img"));
         for (var i = 0, il = images.length; i < il; i++) {
-            if (images[i].getAttribute("data-slimmage") !== null) {
+            if (images[i].getAttribute("data-slimmage") !== null || 
+                (images[i].className && new RegExp("(^|\\s)data-slimmage(\\s|$)").test(images[i].className))) {
```

Comment:
We can do this in a faster/better way when only IE8+ compatibility is needed; 
    window.slimmage.beforeAdjustSrc = function(){
      var to_tag = w.document.querySelector("img.data-slimmage");
      for (var i = 0; il = to_tag.length; i < il; i++){
        to_tag[i].setAttribute("data-slimmage","true");
      } 
    };